content/components/toggle-switch.mdx

```diff
@@ -114,14 +171,19 @@ If you have a group of configuration options that can be disabled, use a button
 
 ![Configuration options section that can be enabled or disabled. Configuration controls are hidden when it is disabled.](https://user-images.githubusercontent.com/2313998/159587120-2d842830-18c7-4429-bd10-f5424f6fac9e.png)
 
-
 <DoDontContainer>
   <Do>
-    <img role="presentation" src="https://user-images.githubusercontent.com/2313998/159587100-1a1af5b2-dd19-456a-a0ed-c2f9fa7f7c2e.png" />
+    <img
+      role="presentation"
```

Comment:
I feel like this image adds important context to the message (my providing examples). We should consider adding alt text here.